src/gtest/test_checktransaction.cpp

```diff
@@ -391,3 +391,115 @@ TEST(checktransaction_tests, non_canonical_ed25519_signature) {
     EXPECT_CALL(state, DoS(100, false, REJECT_INVALID, "bad-txns-invalid-joinsplit-signature", false)).Times(1);
     CheckTransactionWithoutProofVerification(tx, state);
 }
+
+TEST(checktransaction_tests, overwinter_constructors) {
+    CMutableTransaction mtx;
+    mtx.fOverwintered = true;
+    mtx.nVersion = 3;
+    mtx.nFormatBranchId =0xDEADBEEF;
+    // FIXME: Eventually replace with
+    // NetworkUpgradeInfo[Consensus::UPGRADE_OVERWINTER].nBranchId;
```

Comment:
(Posting again elsewhere because the original one shows up as outdated.)

If you rebase on current #2898, this parameter exists 😃 (https://github.com/zcash/zcash/pull/2898/commits/1eac915de98a5cfec9d4db88bdb0eaea91b283ab)